cumulusci/cli/tests/test_cci.py

```diff
@@ -1801,31 +1801,42 @@ def test_task_doc(self, doc_task):
         run_click_command(cci.task_doc, runtime=runtime, project=False)
         doc_task.assert_called()
 
+    def test_task_doc__project__outside_project(self):
+        runtime = mock.Mock()
+        runtime.project_config = None
+        with pytest.raises(click.UsageError):
+            run_click_command(cci.task_doc, runtime=runtime, project=True)
```

Comment:
Added this test to cover the new check for using --project outside of a project directory